packages/api/src/base/Getters.ts

```diff
@@ -21,7 +21,10 @@ interface PkgJson {
 }
 
 function assertResult<T> (value: T | undefined): T {
-  return assertReturn(value, 'Api needs to be initialized before using, listen on \'ready\'');
+  if (isUndefined(value)) {
+    throw new Error('Api needs to be initialized before using, listen on \'ready\'');
+  }
+  return value;
```

Comment:
```suggestion
  if (isUndefined(value)) {
    throw new Error('Api needs to be initialized before using, listen on \'ready\'');
  }
  
  return value;
```

I believe CI will complain here.